.gitignore

```diff
@@ -0,0 +1,8 @@
+.opencm3/
```

Comment:
I would include on documentation from where and how to get opencm3 library to compile the code. Maybe a new issue for that?